internal/characters/wanderer/skill.go

```diff
@@ -0,0 +1,143 @@
+package wanderer
+
+import (
+	"github.com/genshinsim/gcsim/internal/frames"
+	"github.com/genshinsim/gcsim/pkg/core/action"
+	"github.com/genshinsim/gcsim/pkg/core/attributes"
+	"github.com/genshinsim/gcsim/pkg/core/combat"
+	"github.com/genshinsim/gcsim/pkg/core/event"
+	"github.com/genshinsim/gcsim/pkg/core/glog"
+)
+
+const (
+	skillKey = "windfavored-state"
+)
+
+var skillFramesNormal []int
+
+func init() {
+	skillFramesNormal = frames.InitAbilSlice(28)
+
+}
+
+const skillHitmark = 2
+
+func (c *char) skillActivate(p map[string]int) action.ActionInfo {
+	// TODO: Hitlag?
+	c.AddStatus(skillKey, 20*60, true)
+	c.Core.Player.SwapCD = 60 * 14
+
+	// Add 10 seconds worth of skydwellerPoints (1 point = 6 frames)
+	c.skydwellerPoints = 100
+	c.maxSkydwellerPoints = 100
+	c.c6Count = 0
+
+	c.Core.Tasks.Add(c.depleteSkydwellerPoints(), 6)
+
+	// Initial Skill Damage
+	// TODO: Does that even need to be a task?
+	c.Core.Tasks.Add(func() {
+		ai := combat.AttackInfo{
+			ActorIndex: c.Index,
+			Abil:       "Hanega: Song of the Wind",
+			AttackTag:  combat.AttackTagElementalArt,
+			ICDTag:     combat.ICDTagNone,
+			ICDGroup:   combat.ICDGroupDefault,
+			StrikeType: combat.StrikeTypeDefault,
+			Element:    attributes.Anemo,
+			Durability: 25,
+			Mult:       skill[c.TalentLvlSkill()],
+		}
+
+		// TODO: check radius
+		radius := 2.92
```

Comment:
Should be radius 6 and please consider using the number inside of NewCircleHit directly. After that you can remove the TODO.